ahcer/functions/scripts/episodes-update-model.js

```diff
@@ -0,0 +1,119 @@
+const admin = require('firebase-admin');

```

Comment:
Is this a one-and-done script? In other words, should we remove this file at some point in the near future as we won't need to run this again?